src/macros/for_others.rs

```diff
@@ -65,6 +65,68 @@ macro_rules! implement_for_others {
 
 pub(crate) use implement_for_others;
 
+/// Implements [`SetEntry`](crate::traits::SetEntry) for [`*type*`] using the conversions from the
```

Comment:
Please rework your macros in the style of the other macros such that we do not have to refactor it later. The style, its usage and the corresponding comments should be straight forward